diff --git a/node_modules/@safe-global/api-kit/dist/src/utils/config.js b/node_modules/@safe-global/api-kit/dist/src/utils/config.js
index fcf1750..e4f8662 100644
--- a/node_modules/@safe-global/api-kit/dist/src/utils/config.js
+++ b/node_modules/@safe-global/api-kit/dist/src/utils/config.js
@@ -2,6 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.TRANSACTION_SERVICE_URLS = void 0;
 exports.TRANSACTION_SERVICE_URLS = {
+    '5115': 'https://transaction-testnet.safe.citrea.xyz',
     '1': 'https://safe-transaction-mainnet.safe.global',
     '10': 'https://safe-transaction-optimism.safe.global',
     '56': 'https://safe-transaction-bsc.safe.global',
